source/examples/xgboost-azure-mnmg-daskcloudprovider/notebook.ipynb

```diff
@@ -0,0 +1,2433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.            filename=os.path.join(worker.local_directory, model_file), model_type="xgboost"
</pre><p>@jacobtomlinson Forest inference library (FIL) doesn't seem to be loading the xgboost model correctly on the remote dask workers,</p><p>I have tried a few solutions but can't seem to resolve. what am i missing?</p><p> </p><p>^^only thing blocking this PR, all other cell outputs work just fine</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>